applications/FluidDynamicsApplication/custom_python/add_custom_processes_to_python.cpp

```diff
@@ -38,6 +38,7 @@
 #include "custom_processes/move_rotor_process.h"
 #include "custom_processes/mass_conservation_check_process.h"
 #include "spaces/ublas_space.h"
+#include "custom_processes/two_fluids_inlet_process.h"
```

Comment:
We use to alphabetically sort the includes in the FluidDynamicsApplication. Since you will need to further edit the stuff in this PR, would you mind to also take this into account?

Reply:
Ok, I tried to put them in order without destroying the block structure.